### PR TITLE
Allow using render_model_add by passing a class

### LINKED ISSUE #3874: Templatetag "render_model_add" requires a model instance
Allow to use ``render_model_add`` by passing a class as argument instead of  an instance

### PR DESCRIPTION
**Pull request by yakky**
Fix #3874

### CODE REVIEW

**Comment by yakky** [comment]
@mkoistinen you ok with this?

File: cms/tests/templatetags.py [hunk 1]
```diff
@@ -400,7 +400,7 @@ def test_render_model_add(self):
         context = RequestContext(request, {'category': Category()})
         with self.assertNumQueries(0):
             output = template.render(context)
-        expected = 'cms_plugin cms_plugin-sampleapp-category-add-None '
+        expected = 'cms_plugin cms_plugin-sampleapp-category-add-0 '
```
Comment (mkoistinen) [inline cms/tests/templatetags.py]: Awesome, this bothered me a lot. Glad its fixed.

**Comment by mkoistinen** [comment]
Looks like this allows a model class to be passed, which is nice. Does this also allow picking up pre-set values from a passed instance? I can't immediately tell, but the changes to the docs don't mention it.

**Comment by yakky** [comment]
Nope, this is just to add support for class. #3874 has been reverted to just this use case.
The default values use case has been opened separately

**Comment by mkoistinen** [comment]
I vote we apply this to 3.0.x as this is how it was supposed to work from the beginning.

**Comment by yakky** [comment]
Ok, I'll close this and reopen on 3.0.x

**Comment by yakky** [comment]
Supersed by #3899

**Pull request closed without merging**